src/rules/font-weight-notation/index.js

```diff
@@ -76,7 +77,7 @@ export default function (expectation, options) {
       const weightValueOffset = decl.value.indexOf(weightValue)
 
       if (expectation === "numeric") {
-        if (!isNumbery(weightValue)) {
+        if (!isNumbery(weightValue) && weightValue !== INHERIT_KEYWORD) {
```

Comment:
@davidtheclark I think we ignore only on `number` option. Because if use `named-where-possible` option (`inherit` is word) we don't spends time on condition.